chidioguejiofor/maintenance-react-app
# #160038559: set up project

#### What does this PR do?
- adds eslint config to the app
- adds webpack config to the app
- adds babel config for transpiling the app
- creates a basic folder structure to the app

#### Description of Task to be completed?
- set up app dependencies and repo

#### How should this be manually tested?
- clone the repo
- run `npm install`(ensure that you have npm installed in your machine)
- run `npm start` from the app 

#### Any background context you want to provide?
- None
#### What are the relevant pivotal tracker stories?
- 160038559

#### Questions:
-None
